packages/playground/backend-integration/vite.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const glob = require('fast-glob')
+
+/**
+ * @returns {import('vite').Plugin}
+ */
+function BackendIntegrationExample() {
+  return {
+    name: 'backend-integration',
+    config({ root: playgroundRoot }) {
+      const projectRoot = playgroundRoot
+      const sourceCodeDir = path.join(projectRoot, 'frontend')
```

Comment:
I am getting this error in windows
```
error during build:
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:329:5)
    at validateString (node:internal/validators:129:11)
    at Object.join (node:path:397:7)
    at Object.config (C:\Code\p-vite\packages\playground\backend-integration\vite.config.js:12:34)
    at resolveConfig (C:\Code\p-vite\packages\vite\src\node\config.ts:261:27)
    at doBuild (C:\Code\p-vite\packages\vite\src\node\build.ts:308:18)
    at build (C:\Code\p-vite\packages\vite\src\node\build.ts:295:12)
    at CAC.<anonymous> (C:\Code\p-vite\packages\vite\src\node\cli.ts:139:7)
```
Looks like `root` is undefined

Reply:
Thanks, seems like the behavior is different in Windows.

Changed it to use `__dirname` instead, since that works even with `test-build` which creates a copy of the playground.